### DISCUSSION TITLE
Unexpected loading time for Tailwind css with Next.js in Server-side rendering

### DISCUSSION THREAD

**Discussion by annaczinege** [tailwindlabs/tailwindcss#6862] [Help] [answered]
For the website I'm creating I use next.js and for styling tailwind, but can't really see the benefit of server-side rendering that is expected from next.js, because the styles and images have a loading time when you open up the website, (assumingly related to tailwind) and with a slower connection it takes quite a few seconds to get to see the contents of the page.

How could I solve this loading issue and still stick to tailwind?

This is the website link, if you try it, you'll see the few seconds loading time before the content is visible: https://www.revolin.io/

**Comment by Ramos169** [comment]
Thank you for sharing the info, I will try to figure it out more! but first i need to check my [mycardstatement](https://www.mycardstatement.vip/)

**Reply by annaczinege** [reply]
Thanks!

**Comment by alexanderbluhm** [comment, marked as answer]
Hi! Looking at your page load, the CSS file is only a very small part of it. In addition, nearly half of the CSS file size is the `Toastify` plugin you use.  

![Screenshot 2022-01-04 at 10 15 28@2x](https://user-images.githubusercontent.com/6071634/148036601-31d8f2b2-b7fe-4fee-8fd5-be9e9e24fbce.png)

You may have to look at the images and add a _lazy loading_ (**look at the Next.js Image component**).
However, for me the page loading speed looks ok. If you run Google Lighthouse you see some parts where you can improve:   

<img width="767" alt="Screenshot 2022-01-04 at 10 22 25@2x" src="https://user-images.githubusercontent.com/6071634/148037399-432f935a-799b-434f-85b3-090e75c8cf5d.png">

But overall this is no TailwindCSS related topic. Good luck for you! :)

**Reply by annaczinege** [reply]
Thank you very much!
